File: JAVASCRIPT PHASE GATE/SimpleArithmeticApp.js
const prompt = require('prompt-sync')();

const NUMBER_OF_QUESTIONS = 10;

countCorrect = 0;

answer = 0;

countNumberAnswered = 0;

countWrong = 0;

correctAnswer = 0;

        let start = Date.now();

    while(countNumberAnswered != NUMBER_OF_QUESTIONS){
      firstNumber = parseInt(Math.random(0, 10)* 11);
       secondNumber = parseInt(Math.random (0, 10)* 11);

       const correctAnswer = firstNumber - secondNumber;

        if(firstNumber < secondNumber) {
            swap = firstNumber;
            secondNumber = swap;
            firstNumber = secondNumber;
        }

           answer = prompt(firstNumber + " - " + secondNumber + " = ");
            countNumberAnswered += 1;

        if(answer == correctAnswer) {
            countCorrect += 1;
        }

        if(answer != correctAnswer) {
            countWrong += 1;
        }

        if(countWrong == 2){
            break;
        }
        


    }

        stop = Date.now();
        total = stop - start;
//
        console.log("The Number Of Correctly Answered Questions is " + countCorrect);
        console.log("The time taken is " + total/1000 +  " seconds");

File: JAVA PHASE GATE/Day 1/SimpleArithmeticApp.java
import java.util.Scanner;
    public class SimpleArithmeticApp {
        public static void main(String...Odili) {
            Scanner input = new Scanner(System.in);

        
        int startTime = (int)System.currentTimeMillis();

            final int NUMBER_OF_QUESTIONS = 10;
            
            int countCorrect = 0;
            int answer = 0;
            int countNumberAnswered = 0;
            int countWrong = 0;
            int correctAnswer = 0;

        while(countNumberAnswered != NUMBER_OF_QUESTIONS){

        int firstNumber = (int)(Math.random() * 11);
        int secondNumber = (int)(Math.random() * 11);

         correctAnswer = firstNumber - secondNumber;

        if(firstNumber < secondNumber){
            int swap = firstNumber;
            secondNumber = swap;
            firstNumber = secondNumber;
        }

        
        System.out.print(firstNumber +  " - " + secondNumber + " = " );
             answer = input.nextInt();
            countNumberAnswered += 1;

     if(answer == correctAnswer){
        System.out.println("Correct");
           countCorrect += 1;                    
            }
       
        if(answer != correctAnswer){
            System.out.println("Wrong");
            countWrong += 1;
           
        }
          
           
            if(countWrong >= 2) {
               
            }
    }

            int stopTime = (int)System.currentTimeMillis();
            int total = stopTime - startTime;

            System.out.println("The Number Of Correctly Answered Questions is " + countCorrect);

            System.out.println("Time taken is " + total/ 1000 + " seconds");
                

      




        }
    }
